rpc-jobs/RPC-AIO.yml

```diff
@@ -1,82 +1,135 @@
-- job:
-    name: RPC_AIO
+- defaults:
```

Comment:
May be worth putting global defaults in its own file as it will effect all jobs.